projects/storefrontlib/src/cms-components/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.html

```diff
@@ -48,8 +49,18 @@
                   | cxTranslate: { count: cart.deliveryItemsQuantity }
               }}
             </div>
+
             <div>{{ cart.totalPrice?.formattedValue }}</div>
           </div>
+
+          <!-- Promotions -->
+          <div
```

Comment:
Adding new dom elements to existing components is considered a breaking change and should be hidden behind feature level